src/views/BillDisplay/BillDisplay.jsx

```diff
@@ -1,70 +1,131 @@
+            case "transport":
+                return <FaCar color={color} size={24}/>;
+            case "public-transport":
+                return <FaBus color={color} size={24}/>;
+            case "grocery":
+                return <FaShoppingCart color={color} size={24}/>;
+            case "shopping":
+                return <FaShoppingBag color={color} size={24}/>;
+            default:
+                return <FaQuestion color={color} size={24}/>;   
+        };
+    };
+   
+    static filterDateTime = timestamp => {
+        let dateTime = "";
+        const date  = timestamp.split(" ")[0];
+        const time  = timestamp.split(" ")[1];
+        const dateArray = date.split("-");
+        const year = parseInt(dateArray[2]);
+        const month = parseInt(dateArray[1]) - 1;
+        const day = parseInt(dateArray[0]);
+ 
+        const dateCreated = new Date(year, month, day).setHours(0,0,0,0);
+        const todayDate = new Date().setHours(0,0,0,0);
+        if(dateCreated < todayDate){
+            dateTime = (year < new Date(todayDate).getFullYear() ) ? `${day < 10 ? "0"+day : day}/${month+1 < 10 ? "0"+(month+1) : month+1}/${year}` : `${day < 10 ? "0"+day : day}/${month+1 < 10 ? "0"+(month+1) : month+1}`;
+        } 
+        else {
+            const hour = parseInt(time.split(":")[0]);
+            const minute = parseInt(time.split(":")[1]);
+            const AMPM = hour >= 12 ? "PM" : "AM";
+            dateTime = `${(hour > 12 || hour === 0) ? Math.abs(hour - 12) : hour}:${minute} ${AMPM}`;
+        }
+        return dateTime;
     }
-  };
 
-  render = () => {
-    const BillsList = (bills) => (
-      <section>
-        <h1> Bills </h1>
-        <div className="bill__container">
-          {bills.map((bill, key) => (
-            <div key={key} className="bill__card card">
-              <div className="card-body">
-                <h5 className="card-title">{bill.name}</h5>
-                <ul className="bill__list list-group card-text">
-                  <li
-                    className={`bill__list__item list-group-item list-group-item-${this.constructor.billStatusColor(
-                      bill.status
-                    )}`}
-                  >
-                    Bill Status: {bill.status}
-                  </li>
-                  <li className="bill__list__item list-group-item">
-                    Bill category: {bill.category}
-                  </li>
-                  <li className="bill__list__item list-group-item">
-                    Bill balance: ${bill.balance}
-                  </li>
-                </ul>
-              </div>
+    render = () => {
+    /**
+     * @description the search bar of the bill
+     */
+    const searchBill = 
+        <div id="search__bill">
+                <span className="search__icon"><FaSearch /> </span>
+                <input type="text" className="form-control border-0"  placeholder= "Search bill"/>
```

Comment:
As mentioned in the conception, the search bar should filter Bills listed by their name